### PR TITLE
D x S balance patch

### PR DESCRIPTION
**Pull request by chrisinajar**

### CODE REVIEW

**Comment by l34Um1** [comment]
My comments are regarding the individual commits. I made them on the commits, it didn't really work.

File: game/scripts/npc/items/custom/item_dagger_of_moriah.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "sangromancy_self_damage"                         "100 100"
+        "sangromancy_self_damage"                         "75 100"
```
Comment (imaGecko) [inline game/scripts/npc/items/custom/item_dagger_of_moriah.txt]: second level shouldn't really be worse to use than the first level. I would put both at 75

**Comment by imaGecko** [comment]
I would hold off making exceptions for refresher and bkb for the time being. Implementing a purge and/or waiting time after the duel tp would help with the triple ultimate shenanigans, if havingthe item ready every duel still proves to be too strong afterwards then the exceptions can still be introduced at that point

**Comment by chrisinajar** [comment]
Reverted all the things baumi disagreed with.

**Comment by chrisinajar** [comment]
@l34Um1 give it another quick look through to make sure it's good, i should have reverted everything you said no to :+1:

**Comment by l34Um1** [comment]
seems good

**Pull request merged**